sonatype/src/main/scala/org/typelevel/sbt/TypelevelSonatypePlugin.scala

```diff
@@ -101,7 +80,7 @@ object TypelevelSonatypePlugin extends AutoPlugin {
         scalaVersion.value,
         scalaBinaryVersion.value
       ).map { cross =>
-        val host = sonatypeCredentialHost.value
+        val host = "central.sonatype.com"
```

Comment:
I don't think this works.  I'm not sure if they have a way to browse the javadoc online anymore.

Reply:
If this is unset, some cross-module scaladoc references fail the build.  If it's set to nonsense, the build passes ... but links to nonsense.  Removing this default is probably going to break a lot of projects' non-release builds, but there's no _good_ default.

Reply:
I chose the IANA-reserved, cheap, and obviously invalid `javadoc.invalid`.  More people will look at nicely linked release docs than will ever try to follow snapshot docs.